packages/blazor-workspace/NimbleBlazor/Components/NimbleNumberField.razor

```diff
@@ -1,16 +1,17 @@
 ﻿@namespace NimbleBlazor
+@using System.Globalization
 @inherits NimbleInputBase<double?>
 <nimble-number-field
     appearance="@Appearance.ToAttributeValue()"
-    current-value="@CurrentValue"
+    current-value="@CurrentValueAsString"
     @onchange="@(EventCallback.Factory.CreateBinder<string?>(this, __value => CurrentValueAsString = __value, CurrentValueAsString))"
     readonly="@ReadOnly"
     disabled="@Disabled"
     placeholder="@Placeholder"
-    step="@Step"
+    step="@Step.ToString(CultureInfo.InvariantCulture)"
```

Comment:
We should use the new `FormatValueAsString` method everywhere that we format a value as a string, not duplicate its logic.